### DISCUSSION TITLE
Bigger popup window size

### DISCUSSION THREAD

**Discussion by FreePDF64** [okaryo/TabTabTab#204] [Ideas]
I don't know if this is a new feature wish, but I think the TabTabTab users need the possibility to increase the height from TabTabTab to minimum 800px instead of 600px. 
The reason is that most of us certainly use a large monitor with a minimum size of 29 inches. In this case 600px are unfortunately too small.

Thank you for your attention,
Michael

**Comment by FreePDF64** [comment]
Hi,

Is it actually relatively easy to set the height to, for example, 800px? Or is complex programming necessary?

Kind regards,
Michael

**Comment by okaryo** [comment]
Hi Michael,

Thank you for your message. I looked into the specifications of Chrome extensions and found that the size of pop-ups is limited to a width of 800px and a height of 600px. Unfortunately, it's not possible to increase the size of the pop-up beyond these dimensions. As an alternative, I could consider displaying a list of tabs within the settings page. Would this be a useful feature for you?

Best regards,
Ryotaro

**Reply by FreePDF64** [reply]
Hi,

I didn't know the Chrome extension specifications at all... but then everything is fine as it is. Your suggestion is certainly great, but I don't need this implementation. Everything is great! I love the add-on.

Best regards,
Michael

**Comment by FreePDF64** [comment]
Additional note:
In my opinion, the recommendation of max. 600px only says that scroll bars appear/should appear at higher values. Therefore you could also set the height to a maximum of 800px ;) as like other Tab manager extensions for Chromium browsers.

**Reply by okaryo** [reply]
Hi,

I've looked into the Chrome extension documentation and it appears that the maximum height for popup windows is set at 600px.

https://developer.chrome.com/docs/extensions/reference/api/action#:~:text=fit%20its%20contents.-,The,-popup%27s%20size%20must

If you find any extensions with popup sizes exceeding 800px, I'd appreciate it if you could share them as a reference. It might offer some insights.

Thanks!

**Comment by FreePDF64** [comment]
Hi,

Ok - please see as reference:

- https://chromewebstore.google.com/detail/tab-manager-plus-for-chro/cnkdjjdmfiffagllbiiilooaoofcoeff
- https://chromewebstore.google.com/detail/tabli/igeehkedfibbnhbfponhjjplpkeomghi

Kind regards,
Michael

**Reply by FreePDF64** [reply]
But it is your decision. The implementation isn't really important. Please don't put too much work into it. :)

**Reply by okaryo** [reply]
Thanks, I'll take a look!
